rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -29,19 +29,48 @@
 ASSETS_IN_V2_GLOBALDB = 3095
 
 
-def _count_v2_v3_assets_inserted() -> int:
-    """Counts and returns how many assets are to be inserted by globaldb_v2_v3_assets.sql"""
-    assets_inserted_by_update = 0
+def _count_sql_file_sentences(file_name: str, skip_tables: int = 0):
+    """
+    Count the sql lines in scripts used during upgrades. If the skip_tables argument is provided
+    it ignores the [skip_tables first] tables and counts the rows for [the skip_tables + 1] table
+    """
+    insertions_made = 0
+    skipped_tables = 0
     dir_path = Path(__file__).resolve().parent.parent.parent
-    with open(dir_path / 'data' / 'globaldb_v2_v3_assets.sql') as f:
+    with open(dir_path / 'data' / file_name) as f:
+        insertions_made = 0
         line = ' '
         while line:
             line = f.readline()
-            assets_inserted_by_update += 1
+            if skipped_tables < skip_tables:
+                if ';' in line:
+                    skipped_tables += 1
+                continue
+
+            if skipped_tables == skip_tables and 'INSERT' in line:
+                insertions_made = 1
+                continue
+
+            insertions_made += 1
             if ';' in line:
                 break
 
-    return assets_inserted_by_update - 1
+    return insertions_made - 1
+
+
+def _count_v2_v3_assets_inserted() -> int:
+    """Counts and returns how many assets are to be inserted by globaldb_v2_v3_assets.sql"""
+    return _count_sql_file_sentences('globaldb_v2_v3_assets.sql')
```

Comment:
Why make all those different functions with just 1 other function call in the body? Keep it simple and just use the `count_sql_file_sententces` where you need. These functions are not needed

Reply:
It was to make it clearer but will change it and just call `_count_sql_file_sentences`